bot/seasons/christmas/adventofcode.py

```diff
@@ -179,6 +180,10 @@ def __init__(self, bot: commands.Bot):
         """
         Return time left until next day
         """
+        if not is_in_advent():
+            await ctx.send(f"Advent of Code is no longer running this year! Come back next year for more puzzles!")
```

Comment:
Maybe it can just return the time until december 1st of next year?

`The Advent of Code event is not currently running. The next event will start in {time_until_next_event}`

Reply:
Added in 049a42d825fe12fe611783dff5531f4fd05af56c